http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppV2Routes.scala

```diff
@@ -177,6 +177,8 @@ object AppV2Routes {
         ea <- x.downField("extraArgs").as[Option[List[String]]]
         wsi <- x.downField("workspaceId").as[Option[WorkspaceId]]
         swi <- x.downField("sourceWorkspaceId").as[Option[WorkspaceId]]
+        adtm <- x.downField("autodeleteThresholdMin").as[Option[Int]]
```

Comment:
```suggestion
        adt <- x.downField("autodeleteThreshold").as[Option[Int]]
```

Reply:
... also, wasn't sure if you wanted to drop the `d` based on other variables and make it:
```suggestion
        at <- x.downField("autodeleteThreshold").as[Option[Int]]
```